dramatiq/brokers/redis/dispatch.lua

```diff
@@ -75,6 +75,42 @@ for i=8,#ARGV do
     ARGS[i - 7] = ARGV[i]
 end
 
+
+-- In my testing I found the max size that we can unpack to be 7999 (which
+-- is LUAI_MAXCSTACK - 1). Lua can be built with a different LUAI_MAXCSTACK
```

Comment:
Is there any way we can get this exact value at runtime?

Reply:
Unfortunately it's only exposed in lua's C api as far as I can tell. AFAICT, there isn't a way to get those from lua code.